src/utils/components/DetailItem/DetailItem.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  Button,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTermHelpText,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+
+import { DetailItemHeader } from './DetailItemHeader';
+import EditButtonWithTooltip from './EditButtonWithTooltip';
+
+import './DetailItem.scss';
+
+type DetailItemProps = {
+  bodyContent?: ReactNode;
+  breadcrumb?: string;
+  className?: string;
+  'data-test-id'?: string;
+  descriptionData: ReactNode | string;
+  descriptionHeader?: ReactNode;
+  editOnTitleJustify?: boolean;
+  isDisabled?: boolean;
+  isEdit?: boolean;
+  isPopover?: boolean;
+  label?: ReactNode;
+  messageOnDisabled?: string;
+  moreInfoURL?: string;
+  onEditClick?: () => void;
+  showEditOnTitle?: boolean;
+};
+
+const DetailItem: FC<DetailItemProps> = ({
+  bodyContent,
+  breadcrumb,
+  className,
+  'data-test-id': testId,
+  descriptionData,
+  descriptionHeader,
+  editOnTitleJustify = false,
+  isDisabled,
+  isEdit,
+  isPopover,
+  label,
+  messageOnDisabled,
+  moreInfoURL,
+  onEditClick,
+  showEditOnTitle,
+}) => {
+  const { t } = useNMStateTranslation();
+  const NotAvailable = <span className="text-muted">{t('Not available')}</span>;
+
+  const description = (
+    <EditButtonWithTooltip
+      isEditable={!isDisabled}
+      onEditClick={onEditClick}
+      testId={testId}
+      tooltipContent={messageOnDisabled}
+    >
+      {descriptionData ?? NotAvailable}
+    </EditButtonWithTooltip>
+  );
```

Comment:
nested?

Reply:
the DetailItem components was copied from kubevirt-plugin (VirtualMachineDescriptionItem)